exercise-2/exercise-2-10/exercise-2-10.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Volume of a Sphere</title>
+    <style> 
+      body{padding-top:30px;}
+      label,input{display:block;} 
+    </style>
+  </head>
+  <body>
+    <p>Input radius value and get the volume of a sphere.</p>
+    <form action method="post" id="MyForm">
+      <label for="radius">Radius</label>
+      <input type="text" name="radius" id="radius" required>
+      <label for="volume">Volume</label>
+      <input type="text" name="volume" id="volume">
```

Comment:
The value volume input is calculated based on the radius. We should render a read-only input